web/src/beta/features/PluginPlayground/WidgetsList/PropertyItem.tsx

```
@@ -0,0 +1,72 @@
+      default:
+        return "";
+    }
+  });
+
+  const handleChange = (newValue?: string | number | boolean) => {
+    switch (typeof newValue) {
+      case "string":
+        setValue(newValue);
+        break;
+      case "number":
+        if (!isNaN(newValue)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)